components/devtools/actors/network_event.rs

```diff
@@ -366,11 +370,24 @@ impl NetworkEventActor {
 
     pub fn event_actor(&self) -> EventActor {
         // TODO: Send the correct values for startedDateTime, isXHR, private
+        let duration = self
+            .request
+            .startedDateTime
+            .duration_since(UNIX_EPOCH)
+            .unwrap_or_default();
+
+        let started_datetime_rfc3339 =
+            match Local.timestamp_opt(duration.as_secs() as i64, duration.as_nanos() as u32) {
```

Comment:
I think you are now including the duration twice when calculating the local time. Nanoseconds should only include the portion of the time that isn't represented by the seconds part.

Reply:
I see, I can use `timestamp_millis_opt` and provide `millis`, as you mentioned that millisecond accuracy would be a better fit here."